src/platform/packages/private/kbn-screenshotting-server/src/paths.ts

```diff
@@ -103,7 +103,7 @@ export class ChromiumArchivePaths {
   ];
```

Comment:
I'm slightly concerned this might impact backports of the updated checksums for chromium during the puppeteer upgrade. Leaving this comment for posterity sake, I'll confirm and leave an update as to how it might impact our backports

Reply:
@eokoneyo @Dosant I think this PR is going to be backported to 8.x, so we should be OK.

Reply:
@tsullivan I was thinking more when we have to backport to 7.x. There's one due for this month so I suppose we'll find out what the impact might be.